plugins/hls-brittany-plugin/hls-brittany-plugin.cabal

```diff
@@ -0,0 +1,69 @@
+cabal-version: 2.2
+name:          hls-brittany-plugin
+version:       1.0.0.0
+synopsis:      Integration with the Brittany code formatter
+license:       Apache-2.0
+license-file:  LICENSE
+author:        The Haskell IDE Team
+copyright:     The Haskell IDE Team
+maintainer:    alan.zimm@gmail.com
+category:      Development
+build-type:    Simple
+extra-source-files:
+  LICENSE
+
+library
+  exposed-modules:  Ide.Plugin.Brittany
+  hs-source-dirs:   src
+  build-depends:    base
+               ,    brittany      >= 0.13.1.0
+               ,    filepath
+               ,    ghc
+               ,    ghc-boot-th
+               ,    ghcide
+               ,    lsp-types
+               ,    hls-plugin-api
```

Comment:
I guess we also need version bounds for `ghcide` and `hls-plugin-api`.

Reply:
Same thing, I'm not convinced that the plugins need artificial version bounds. Is this approach documented anywhere? Rationale and tradeoffs?

Reply:
I think at least we need to be consistent in versioning. If you think it would be better to not give such artificial version bounds even for this 1.0.0.0 release, I think we at least need to remove version bounds I added in #1402 in order to be consistent.

Reply:
Is the bound artificial? I'm pretty sure it will only build with `ghcide` and `hls` 1.0 because of the lsp upgrade.

Reply:
ghcide 0.7.5 already has the lsp upgrade

Reply:
> I think we need to be consistent in versioning. If you think it would be better to not give such artificial version bounds even for this 1.0.0.0 release, I believe that we, at least, need to remove version bounds I added in #1402 in order to be consistent.

Consistency is not a good argument for an approach that deviates from the standard. What I am missing is a *document* that explains the approach, which packages it applies to (hls, hls-plugin-api, ghcide, plugins, etc) and, if possible, the motivation and trade-offs for the approach. Without a document, I can see many future questions for clarification on version bounds and endless disagreements.

Reply:
I mean, if you see this as a "deviation from standard", that perspective should be applied to other packages too. I think that level of consistency should make sense, not as an argument for "this package should have ^>= 1.0.0.0 bound", but for "Haskell-language-server packages should use a unified versioning policy".

Reply:
I am asking: what is the haskell language server versioning policy? 
Since it's clearly not the PVP which is well known, it deserves being documented somewhere.

Reply:
I thought the point of 1.0.0 release is following PVP after 1.0.0 and upward? If I'm wrong, please correct me.